src/Traits/SocialLoginTrait.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Canvas\Traits;
+
+use function Canvas\Core\envValue;
+use function time;
+use Canvas\Models\Sources;
+use Canvas\Models\Users;
+use Canvas\Models\UserLinkedSources;
+use Phalcon\Security\Random;
+
+/**
+ * Trait SocialLoginTrait.
+ *
+ * @package Niden\Traits
+ */
+trait SocialLoginTrait
+{
+    /**
+     * Facebook Login
+     * @param Sources $source
+     * @param string $accessToken
+     * @return Array
+     */
+    protected function facebookLogin(Sources $source, string $accessToken): array
```

Comment:
@rwhite27 if the trait is named social login why name the function facebook login? isnt it better just facebook?